Intersect.Client/Interface/Game/EscapeMenu.cs

```diff
@@ -100,80 +69,44 @@ public override void Invalidate()
         }
 
         base.Invalidate();
-        if (Interface.GameUi != null && Interface.GameUi.GameCanvas != null)
+        if (Interface.GameUi?.GameCanvas != null)
         {
             Interface.GameUi.GameCanvas.MouseInputEnabled = false;
```

Comment:
This line can be removed, as discussed in Discord it has no effect and isn't necessary.